RELEASE_NOTES.md

```diff
@@ -1,8 +1,52 @@
 # Release Notes
 
+## stable-1.15-2.4.0, stable-1.16-2.4.0, stable-1.17-2.4.0
+
+* Istio:
```

Comment:
Should mention that istio got upgraded to 1.6.8 as that is why all of these changes got pulled in

Reply:
It should be at 1.6.9 for this release now, yes?

Reply:
It's what's in the release notes. /shrug